app/api/services/recipesService.js

```diff
@@ -1,5 +1,20 @@
 import { Recipe } from '../models/recipes.js';
 import { User } from '../models/users.js';
+import { sequelize } from '../db/db.js';
+import { QueryTypes } from 'sequelize';
+
+async function listRecipes(limit, offset, whereCondition = null) {
```

Comment:
може є сенс додати дефолтні значення тут?
```suggestion
async function listRecipes(limit = 12, page = 1, whereCondition = null) {
    const offset = (page - 1) * limit;
```